superset/db_engine_specs/bigquery.py

```diff
@@ -282,3 +303,33 @@ def df_to_sql(
                 to_gbq_kwargs[key] = to_sql_kwargs[key]
 
         pandas_gbq.to_gbq(df, **to_gbq_kwargs)
+
+    @classmethod
+    def build_sqlalchemy_url(cls, parameters: BigQueryParametersType) -> str:
+        project_id = (
+            parameters.get("credentials_json", {})
+            .get("credentials_info", {})
+            .get("project_id")
+        )
+
+        return f"{cls.drivername}://{project_id}"
```

Comment:
One thing we might consider this is that we would have to pass the credentials info in two different places: `secure_extra` and `parameters`. We need to pass it in `secure_extra` so it's stored encrypted in the database (in the `encrypted_extra` column). But I think this is fine for now.

Reply:
Also, @eschutho, should we use brackets or triple slashes here? I remember you had problems with this before.